### PR TITLE
[MRG] new tests: delete foreign messages + search

### PR DESCRIPTION
**Pull request by gedankenstuecke**
The tests for the search were just commented out, fixed those. Also added the missing test from #394.

### CODE REVIEW

**Comment by gedankenstuecke** [comment]
Added two more tests as I was at it. 😂

File: spec/features/search_spec.rb [hunk 1]
```diff
@@ -3,43 +3,35 @@
...
-    #  click_on('Papers')
-    #  expect(find(:css, '#snpedia-papers')).to have_content('bar foo')
-    #end
+    visit '/'
+    fill_in 'search', with: 'foo'
+
+    form = page.find('form')
+    class << form
+      def submit!
+        Capybara::RackTest::Form.new(driver, native).submit({})
```
Comment (philippbayer) [inline spec/features/search_spec.rb]: Ah beautiful! You figured it out, nice

Comment (gedankenstuecke) [inline spec/features/search_spec.rb]: You don't want to know how much time I spent to figure that one out 😂

Comment (philippbayer) [inline spec/features/search_spec.rb]: I gave up after an hour so I can imagine :D

Comment (gedankenstuecke) [inline spec/features/search_spec.rb]: I saw your failed try with \n ;)

**Comment by tsujigiri** [comment]
👏

**Comment by gedankenstuecke** [comment]
@tsujigiri: I'll totally turn into you soon when it comes to testing 😂😂

**Comment by tsujigiri** [comment]
Spooky. :ghost: 

It's just what I'm used to from work. It greatly reduces the fear of shipping things several times a day. :smile:

**Comment by gedankenstuecke** [comment]
Yes, I gotta say I sleep much sounder since we are seriously trying to increase our test coverage here. 👍  Also want to get us over the 80% mark, that's why it would be awesome if you could tell me what I'm doing wrong in #398 😉

**Pull request merged**